src/table/body-cell/index.tsx

```diff
@@ -56,6 +57,21 @@ function TableCellEditable<ItemType>({
   const [hasFocus, setHasFocus] = useState(false);
   const showIcon = hasHover || hasFocus;
 
+  const prevSuccessfulEdit = usePrevious(successfulEdit);
+  const prevHasFocus = usePrevious(hasFocus);
+  const [showSuccessIcon, setShowSuccessIcon] = useState(false);
+
+  useEffect(() => {
+    // Hide the success icon after a successful edit, when cell loses focus.
+    if (successfulEdit && prevSuccessfulEdit && !hasFocus && prevHasFocus) {
+      setShowSuccessIcon(false);
```

Comment:
This line is covered by this [newly added integ test case](https://github.com/cloudscape-design/components/pull/1264/files#diff-21cfe5e571a29f732cbfb7d3b2cf38c294086bbf84d2c047d1e61b14fefa2fbdR154-R177).

Reply:
Got it. I think we want to avoid disabling codecov checks in GitHub. So we have two options:

1) Add an inline comment here to ignore the line in codecov
2) Find a way to test this in a unit test (preferred)

Reply:
Makes sense, I added a unit test for this case. I also added more unit test scenarios that don't have to be integration tests and would run much faster. Codecov is happy now.